### PR TITLE
feat: LT-2718 - Display all product details

### PR DESCRIPTION
**Pull request by iichr**
- Display all good category details in a popup on clicking View or on a good in the sidebar when reviewed
  - Popup and related styling etc kindly provided by @jfaracik - kudos!

Screenshot 2020-07-08 at 12 23 09

### CODE REVIEW

File: templates/layouts/case.html [hunk 1]
```diff
@@ -187,6 +187,36 @@ <h1 id="heading-reference-code" class="app-case-header__heading">
...
+				top=${top},
+				left=${left}
+				`
+			)
+
+			if (window.focus) newWindow.focus();
+		}
+
+		function openGoodModal(goodId) {
+			popupCenter({url: `/queues/{{ queue.id }}/cases/{{ case.id }}/good/${goodId}/`, title: 'xtf', w: 300, h: 500});
```
Comment (jfaracik) [inline templates/layouts/case.html:216]: Title should be set to the good name

File: templates/case/slices/goods.html [hunk 2]
```diff
@@ -113,12 +114,16 @@ <h2 class="govuk-heading-m">{{ case.goods|length }} good{{ case.goods|pluralize
...
+						<td class="govuk-table__cell {% if show_advice %}lite-!-no-border{% endif %}">
+							<a class="govuk-link govuk-link--no-visited-state" onclick="this.style.backgroundColor='yellow'; return openGoodModal('{{ good.good.id }}')">View</a>
+						</td>
 					</tr>
 					{% if show_advice %}
 						{% include 'includes/advice.html' with id=good.good.id %}
 					{% endif %}
 				{% endfor %}
 			</tbody>
+
```
Comment (jfaracik) [inline templates/case/slices/goods.html:126]: Empty line :)!

File: templates/case/slices/destinations.html [hunk 3]
```diff
@@ -116,4 +116,4 @@ <h2 class="govuk-heading-m">{{ destinations|length }} destination{{ destinations
 			</form>
 		{% endif %}
 	{% endif %}
-{% endwith %}
+{% endwith %}
```
Comment (jfaracik) [inline templates/case/slices/destinations.html:119]: Would you mind adding a closing line so this file doesn't show in the PR?

Comment (iichr) [reply templates/case/slices/destinations.html:119]: yes will do

File: templates/layouts/case.html (hunk 1 above)
Comment (iichr) [reply templates/layouts/case.html]: Changed

**Pull request merged**